### ISSUE TITLE
MP Starter Project needs historic documentation

### ISSUE THREAD

**Issue by aeiras** [microprofile/microprofile-marketing#32]
[direct twitter feedback](https://twitter.com/danielbryantuk/status/1098862902205927424)

**Comment by rdebusscher** [comment]
@aeiras Please open issue in MicroProfile Starter repo so that all issues are centralized.

What is expected from this historic documentation (release notes?)

**Comment by aeiras** [comment]
great feedback @rdebusscher :) 

Yes Rudy. Simple tracing is what is needed. For instance, I remember:

- the very first time you attending the MP community call and we asked you to show us what you had for starter during that call. It was not even a part of the call agenda yet because you #showup, you made it possible. 

- from them, the very first Starter thread via forum was sent... 

- that lead to the creation of the Starter Repo

In short, what we want to document is how you as an individual contributor back then was able to bring much value to MicroProfile. NOT only by showing up but most importantly by following through and collaborating via open channels. 

See-- no need to glamour, use links from forum as much as possibe too.

Have fun and thanks for being awesome! I will close this ticket. :)
